conans/client/tools/oss.py

```diff
@@ -71,6 +71,14 @@ def detected_architecture():
     elif "arm" in machine:
         return "armv6"
 
+    # processor reports powerpc on AIX
+    processor = platform.processor()
+    if "powerpc" in processor and OSInfo().is_aix:
+        # query bitness by using getconf
+        kernel_bitness = OSInfo().getconf("KERNEL_BITMODE")
```

Comment:
currently detect_architecture relies on the bitness of python and not on bitness of the system its running on, so probably I will change it to that behavior too